docs/index.html

```diff
@@ -2,69 +2,191 @@
 layout: default
 title: eBay Skin
 ds: 6
+ds_map:
+    alert-dialog:
+        name: alert-dialog
+        version: 2.1
+    avatar:
+        name: avatar
+        version: 1.0
+    badge:
+        name: badge
+        version: 1.2
+    breadcrumbs:
+        name: breadcrumb
+        version: 1.2
+    button:
+        group: button
+        name: button
+        version: 1.6
+    carousel:
+        name: carousel
+        version: 1.1
+    calendar:
+        name: calendar
+        version: 1.0
+    checkbox:
+        name: checkbox
+        version: 2.0
+    chip:
+        version: 1.0.0
+    chips-combobox:
+        name: chip-combobox
+        version: 1.0.0
+    combobox:
+        name: dropdown
+        version: 2.2
+    confirm-dialog:
+        name: alert-dialog
+        version: 2.1
+    cta-button:
+        group: button
+        name: button
+        version: 1.2
+    date-textbox:
+        name: date-picker
+        version: 1.0
+    details:
+        name: expansion
+        version: 1.2
+    drawer-dialog:
+        name: bottom-sheet
+        status: deprecated
+        version: 2.0
+    eek:
+        name: eek
+        version: 1.1
+    filter-button:
+        name: quick-filter
+        version: 2.0
+    filter-menu:
+        version: 2.0
+    filter-menu-button:
+        name: quick-filter
+        version: 2.0
+    flag:
+        version: 1.0
+    floating-label:
+        name: form-field
+        version: 2.2
+    fullscreen-dialog:
+        name: modal
+        version: 2.1
+    icon:
+        name: icon
+        version: 1.3
+    icon-button:
+        group: button
+        name: icon-button
+        version: 1.6
+    infotip:
+        name: infotip
+        version: 2.0
+    inline-notice:
+        name: alert-notice
+        version: 2.2
+    lightbox-dialog:
+        - name: modal
```

Comment:
Since this schema organizes mixed information for/from Skin as well as "DS," it can be a bit confusing. Since this is a schema that lives in Skin, unless there are distinctions between the mixed props, it would be natural for devs to assume this is all for Skin. Can we rename the data props specific to DS? It would provide strong inline indication to help distinguish the data.

Can we rename `name` to `ds-name` ?

Reply:
Absolutely. I'm changing to `ds-name`, `ds-group`, and `ds-version`. @ArtBlue, any thoughts on whether we should keep `ds_map` or name it more explicitly `skin_to_ds_map`?

Reply:
Awesome! Thanks! I think `ds_map` is fine since this is a file within Skin and the inference should be clear.